spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/thymeleaf/ThymeleafAutoConfiguration.java

```diff
@@ -159,18 +152,6 @@ public LayoutDialect layoutDialect() {
 	}
 
 	@Configuration
-	@ConditionalOnClass(DataAttributeDialect.class)
```

Comment:
DataAttributeDialect 2.x supports thymeleaf 3.0 now. Will it be removed in spring boot 1.4?